fusionSuite/backend
# Add changeslogs

Changes proposed in this pull request:

- add changelogs with message, old value, new value...

How to test the feature manually:

1. update a property, type item and you will have property changeslgos in the JSON (only get one, not when get all)



Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._

Related to #
